lib/node-utils.ts

```diff
@@ -253,3 +254,42 @@ export function getImportModuleName(
     return node.arguments[0].value;
   }
 }
+
+type AssertNodeInfo = {
+  matcher: string | null;
+  isNegated: boolean;
+};
+/**
+ * Extracts matcher info from MemberExpression node representing an assert.
+ */
+export function getAssertNodeInfo(
```

Comment:
This was extracted from the rule itself to get info about the assert node. Could be useful in other rules if assert statement impacts the rule reporting.